rtos/rtx2/mbed_rtx_conf.h

```diff
@@ -32,7 +32,10 @@
 #endif
 #endif
 
-#define OS_DYNAMIC_MEM_SIZE         0
+/* mbed OS uses this macro as an on/off switch for RTX dynamic memory. The value has to be either 0 to disable
+   or any other number (dividable by 8) to enable.
+ */
+#define OS_DYNAMIC_MEM_SIZE         8
```

Comment:
Yeah. On the second thought, I may as well contact RTX guys to finish what they started and provide separate way of switching it on and off.